### PR TITLE
[BIOMAGE-1243] - Change user pool name

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-1243

#### Link to staging deployment URL 

https://ui-agi-cognito-test.scp-staging.biomage.net/

#### Links to any Pull Requests related to this

UI - https://github.com/biomage-ltd/ui/pull/592
API - https://github.com/biomage-ltd/api/pull/273

#### Anything else the reviewers should know about the changes here

Change user pool name to use case-insensitive user pool

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Code updates
Have best practices and ongoing refactors being observed in this PR

- [ ] Migrated any selector / reducer used to the new format
- [ ] Validated that current unit tests work as expected and are enough

### Testing
- [ ] Unit tests written
- [ ] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [ ] Deployed to staging
- [ ] Passed integration tests 

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/utils/ssr/getAuthenticationInfo.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ const getAuthenticationInfo = async () => {
    * local development.
    */
   const k8sEnv = process.env.K8S_ENV || 'staging';
-  const userPoolName = `biomage-user-pool-${k8sEnv}`;
+  const userPoolName = `biomage-user-pool-insensitive-${k8sEnv}`;
```
Comment (ivababukova) [inline src/utils/ssr/getAuthenticationInfo.js]: what do you mean by "insensitive"?

Comment (aerlaut) [reply src/utils/ssr/getAuthenticationInfo.js]: "case-insensitive"

Comment (aerlaut) [reply src/utils/ssr/getAuthenticationInfo.js]: Is it ok? or does it make it ambiguous?

**Review by ivababukova — APPROVED** [review]

File: src/utils/ssr/getAuthenticationInfo.js (hunk 1 above)
Comment (aerlaut) [reply src/utils/ssr/getAuthenticationInfo.js]: changed

**Pull request merged**